index.html

```diff
@@ -59,133 +67,197 @@ <h2 class="main-subtitle">I'am a Software Developer</h2>
 
 		<!-- Works Section -->
 		<section class="work-section">
-			<h2 class="work-title">My Recent Works</h2>
-			<div class="work-bar"></div>
+			<div class="work-text-container">
+				<h2 class="work-title">My Recent Works</h2>
+				<div class="work-bar"></div>
+			</div>
 			<!-- Work Card Section -->
 			<div class="work-card">
-				<img src="media/multi-post-stories.png" alt="Multi-Post Stories Image" class="card-image">
-				<h3 class="card-title">Multi-Post Stories</h3>
-				<p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.
-					Has been the industry's standard dummy text ever since the 1500s,
-					when an unknown printer took a standard dummy text.</p>
-				<div>
+				<div class="work-card-img">
+					<img src="media/multi-post-stories.png" alt="Multi-Post Stories Image" class="card-image">
+				</div>
+				<div class="work-card-text">
+					<h3 class="card-title">Multi-Post Stories</h3>
+					<p class="card-text">A daily selection of privately personalized reads; no accounts or sign-ups required.
+						Has been the industry's standard dummy text ever since the 1500s,
+						when an unknown printer took a standard dummy text.</p>
 					<ul class="btn-container">
 						<li><button class="feature-btn" type="button">css</button></li>
 						<li><button class="feature-btn" type="button">html</button></li>
 						<li><button class="feature-btn boostrap" type="button">boostrap</button></li>
 						<li class="ruby"><button class="feature-btn" type="button">Ruby</button></li>
 					</ul>
+					<a href="#" rel="noopener noreferrer"><button class="project-btn" type="button">See Project</button></a>
 				</div>
-				<a href="#" rel="noopener noreferrer"><button class="project-btn" type="button">See Project</button></a>
 			</div>
 
 			<!-- Projects Cards -->
-			<div class="card-container">
-				<div class="card-content">
-					<h3 class="project-title">Profesional Art Printing Data</h3>
-					<p class="project-text">A daily selection of privately personalized reads; no accounts or sign-ups required.
-						has been the industry's standard.</p>
-					<div>
-						<ul class="project-btn-container">
-							<li class="html-list"><button class="project-feature-btn" disabled="disabled" type="button">html</button>
-							</li>
-							<li><button class="project-feature-btn boostrap" disabled="disabled" type="button">boostrap</button></li>
-							<li class="project-ruby"><button class="project-feature-btn" disabled="disabled"
-									type="button">Ruby</button></li>
-						</ul>
+			<!-- First Cards -->
+			<div class="cards-container">
+				<div class="card-container card-1">
+					<div class="card-content card-1-content">
+						<h3 class="project-title">Profesional Art Printing Data</h3>
+						<p class="project-text">A daily selection of privately personalized reads; no accounts or sign-ups required.
+							has been the industry's standard.</p>
```

Comment:
- [ ] This section currently is not shown correctly, I can only see one card and it's not displayed correctly and doesn't have the required interactions (text and dark opacity should disappear upon cursor hover), please refer to the images below 🔽 

### Figma Design
![Figma cards](https://user-images.githubusercontent.com/104319462/220787001-2d991abe-b80c-412a-84b1-de8395d8ba24.png)
### Your Design
![your cards](https://user-images.githubusercontent.com/104319462/220787024-f8b4fc70-be25-4388-851c-c4857cdc07c3.png)